packages/select/src/FilterMultiSelect/components/Root/Root.tsx

```diff
@@ -48,9 +47,9 @@ export type FilterMultiSelectProps = RootProps
 export const Root: React.VFC<RootProps> = ({
   trigger,
   children,
-
-  defaultOpen = false,
-
+  isOpen,
+  defaultOpen,
```

Comment:
Had a discussion with @OliBridgman and now that we allow the controlled option, we should remove the default value for `defaultOpen` since the internals in react-aria treats `false` and `undefined` differently (`defaultOpen` should only be defined if it's uncontrolled, but having it always defined could potentially override `isOpen`'s `false` if it were defined even though it should be controlled).